packages/server/lambdas/alfresco/delete-file.ts

```diff
@@ -0,0 +1,62 @@
+import { CategoryDataBase } from '@prisma/client';
+import { ALBEvent, ALBResult } from 'aws-lambda';
+import { findEndpoint } from '../../utils/alfresco';
+import { getRataExtraLambdaError, RataExtraLambdaError } from '../../utils/errors';
+import { log } from '../../utils/logger';
+import { getUser, validateReadUser, validateWriteUser } from '../../utils/userService';
+import { DatabaseClient } from '../database/client';
+import { deleteFileRequestBuilder } from './fileRequestBuilder';
+
+const database = await DatabaseClient.build();
+
+let fileEndpointsCache: Array<CategoryDataBase> = [];
+
+/**
+ * Update custom content for page. Example request: /api/alfresco/file/linjakaaviot/test.pdf
```

Comment:
Should probably say `Remove custom content for page.` or something to that effect.